README.md

```diff
@@ -192,6 +193,13 @@ Unless a group or event identifies as political in nature, misconduct includes d
 Notwithstanding, discussions about _policies_ are generally acceptable. There is a difference between discussing policies in the context of their impact on the community and sharing reactionary comments to recent headlines. While it is acceptable to share reactions generally, reactions to political headlines or controversial topics are only accepted when a group, event, or space identifies as political in nature.
 
 
+### Disrespect of religion or spirituality
```

Comment:
Structure of both these paragraphs is "Don't do this, but it's okay to do this" which reads a little strange to me. Might be simpler to start with a list of "Good behaviors" and then put "Bad behavior" second (sort of in keeping with our overall rewrite to start with positive things before going through explicit lists of bad behavior)?

Reply:
Good point. We have this structure elsewhere (e.g. politics). For religion, I do think it's better to start with acceptable...